src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,184 @@
+                                         cluster), if not specified subctl will try to discover this detail, and if
+                                         it's unable to discover it will prompt the admin.
+
+    --colorcodes string                  color codes (default "blue"), coma separated values
+                                         please use with caution, as it may change in the future. This setting is
+                                         used to make submariner gateway form connections only with clusters
+                                         which share the same color code. This will be replaced by something
+                                         more flexible in the future.
+
+    --no-label                           skip gateway labeling
+                                         this will disable prompting the administrator for a worker node to use
+                                         as gateway.
+
+    --subm-debug                         Enable Submariner debugging (verbose logging)
+
+    --disable-cvo                        disable OpenShift's cluster version operator if necessary, without prompting
+                                         currently, lighthouse service modifies the cluster dns operator in OpenShift
+                                         and we need to disable CVO, this setting disables the prompt which warns the
+                                         administrator.
+
+##### join flags (globalnet)
+
+    --globalnet-cluster-size uint        Cluster size for GlobalCIDR allocated to this cluster (amount of global IPs).
+
+    --globalnet-cidr string              GlobalCIDR to be allocated to the cluster, this setting is exclusive with
+                                         --globalnet-cluster-size and configures a specific globalnet CIDR for this
+                                         cluster.
+
+##### join flags (IPSec)
+
+    --disable-nat                        Disable NAT for IPsec.
+
+    --ikeport <value>                        IPsec IKE port (default 500)
+
+    --ipsec-debug                        Enable IPsec debugging (verbose logging)
+
+    --nattport int                       IPsec NATT port (default 4500)
+
+##### join flags (images and repositories)
+
+    --repository string                  image repository, the repository from where the various submariner images
+                                         will be sourced. (default "quay.io/submariner")
+
+    --version string                     image version
+
+    -o, --operator-image string          the operator image you wish to use
+                                         (default "quay.io/submariner/submariner-operator:$version")
+
+    --service-discovery-repo string      Service Discovery Image repository (default "quay.io/submariner")
+
+    --service-discovery-version string   Service Discovery Image version
+
+#### info
+
+`subctl info broker-info.subm [flags]`
```

Comment:
Do we really require broker-info.subm? 
I dont have an active cluster to verify, but I was using the command without passing broker-info.subm